test/Microsoft.NET.Sdk.Functions.Generator.Tests/HasUnsupportedAttributesTests.cs

```diff
@@ -9,6 +10,11 @@ namespace Microsoft.NET.Sdk.Functions.Test
 {
     public class HasUnsupportedAttributesTests
     {
+        public HasUnsupportedAttributesTests()
+        {
+            AssemblyLoadContext.Default.EnterContextualReflection();
```

Comment:
What is this for?

Reply:
A while back this was added -- https://github.com/Azure/azure-functions-vs-build-sdk/pull/360. 

So we now use the contextual LoadContext to figure out which assemblies to load. But the tests were never updated to take this into account -- so the contextual LoadContext is null in tests b/c it was never initialized. And since we didn't run tests in CI, we didn't notice this.